docs/source/private_data_tutorial.rst

```diff
@@ -1083,6 +881,79 @@ If you go back and look at the block height in your logs, you will see that the
 due to hitting the fourth block (we only increased by two blocks in transferring the asset to Org2), but rather
 because when the asset transfer occurred, Org1 did not meet the ``read`` policy of the ``Org2MSPPrivateCollection``.
 
+
+.. _pd-purge:
+
+Purge Private Data
+------------------
+
+For use cases where private data only needs to be on the ledger until it can be
+replicated into an off-chain database, it is possible to "purge" the data after
```

Comment:
replication to an off-chain database is one use case, but there are other use cases. So perhaps say something more general like, e.g.
"In some cases private data only needs to be persisted for a short time, for example while a multi-party business process is in progress, or until the data is replicated into an off-chain database"...

Reply:
For use cases where private data only needs to be persisted for a short period of time,
it is possible to "purge" the data after a certain set number of blocks, leaving
behind only hash of the data that serves as immutable evidence of the transaction.
An organization could decide to purge private data if the data contained sensitive
information that was used by another transaction, but is not longer needed, or
if the data is being replicated into an off-chain database.